src/napari/_vispy/visuals/interaction_box.py

```diff
@@ -105,3 +111,15 @@ def set_data(
             )
         else:
             self.markers.set_data(pos=np.empty((0, 2)))
+
+    def _compute_bounds(self, axis, view):
+        bounds = None
+        with suppress(ValueError):
+            for v in view._subvisuals:
+                if v.visible:
```

Comment:
We need a comment about *what* is being suppressed, when it can happen, and why it't not handled but just suppressed. Also, can this suppress be restricted to a smaller portion of code?

Reply:
Will add a comment regarding < 1 zoom. It actually only happens in 3-D cases when the zoom box is quite large (>50% of the canvas).

Reply:
> We need a comment about _what_ is being suppressed, when it can happen, and why it't not handled but just suppressed. Also, can this suppress be restricted to a smaller portion of code?

Good point - I remember having to add this suppression a while back in my own codebase, but helpfully I didn't leave a comment for myself when it happened. I am gonna try to reproduce it and will add a comment here.